services/data_deletion/clean_archived_data.py

```diff
@@ -0,0 +1,22 @@
+from helpers.database import database_uri
+from sqlalchemy import create_engine, MetaData, and_
+from datetime import timedelta, datetime
+import celery
+
+
+@celery.task(name='clean_archived_data.delete_archived_data')
+def delete_archived_data():
+    database_engine = create_engine(database_uri)
```

Comment:
would you please put in a docstring which explains what this function, specifically to delete data which are 30 days old with also an archived state.